src/main/java/io/github/kawamuray/wasmtime/wasi/WasiConfig.java

```diff
@@ -19,5 +22,15 @@
     public WasiConfig(String[] args, PreopenDir[] preopenDirs) {
         this.args = args;
         this.preopenDirs = preopenDirs;
+        this.stdoutFile = null;
```

Comment:
can you make this constructor to call the another one like below to ensure all construction process remain consistent?

```java
this(args, preopenDirs, null, null)
```